python/lsst/ts/gis/commander.py

```diff
@@ -135,21 +140,13 @@ def get_raw_string(self, reply):
 
         Returns
         -------
-        raw_status : `bytearray`
-            The unfiltered response of the reply in a byte array.
+        bit_status : `str`
```

Comment:
The method returns a `bytearray` now and no longer a string. Please adjust the description of the parameter as well.